phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widgets/lists/icons/IconsListElement.java

```diff
@@ -80,13 +81,14 @@ public IconsListElement(E item, URI miniatureUri, String title, boolean enabled)
             this.setClip(clip);
         });
 
-        final Label label = new Label(title);
-        label.getStyleClass().add("miniatureText");
+        final Region miniature = new Region();
```

Comment:
My idea was to just copy+paste the three URIs into the `Theme` or `ThemeManager` and delete the `StaticMiniature` because it's unused.